.gitattributes

```diff
@@ -0,0 +1,105 @@
+# These settings are for any web project
```

Comment:
I'm not familiar with this file. I kind of feel like we probs don't need it since it's not specific to firebase functions.